zc_common/remote_resource/parsers.py

```diff
@@ -100,3 +100,49 @@ def parse(self, stream, media_type=None, parser_context=None):
 
         else:
             raise ParseError('Received document does not contain primary data')
+
+
+class ResourceDataParser(object):
```

Comment:
What is the need for this new class? As I mentioned [here](https://github.com/ZeroCater/mp-users/pull/54#discussion_r83147537) I think it's better to keep this in the consumer. I don't think there is always a guarantee that the publisher of the event and the consumer will match the resources structure exactly. This also couples the publisher and consumer, something we would like to avoid.